src/main/java/cz/brmlab/yodaqa/pipeline/structured/StructuredPrimarySearch.java

```diff
@@ -311,4 +329,43 @@ protected void clueAnswerFeatures(AnswerFV afv, Clue clue) {
 	public int getCasInstancesRequired() {
 		return MultiThreadASB.maxJobs * 2;
 	}
+
+	/** Possibly create files for python training, one file per question. */
+	protected synchronized void createPropertyLabels(JCas questionView, PropertyValue p, JCas jcas){
+//		System.setProperty("cz.brmlab.yodaqa.property","data/jacana-property-train");
+		String jacana = System.getProperty("cz.brmlab.yodaqa.property");
```

Comment:
This sounds like a very non-descriptive property name. What about cz.brmlab.yodaqa.dump_property_labels ?